Dockerfile

```diff
@@ -1,7 +1,16 @@
-############# BUILDTIME #############
-# BuildTime Image
+############# BUILDER #############
+# Builder Image
 FROM rustlang/rust:nightly AS Builder
 
+ARG MASSA_GIT_BRANCH="testnet_19"
+ARG MASSA_GIT_REPO="https://github.com/massalabs/massa.git"
+ARG CARGO_BUILD_JOBS=4
```

Comment:
It's not equal to 8 in the .env file ?

Reply:
These are default arguments if not provided